lib/cisco_node_utils/cmd_ref/acl_v4.yaml

```diff
@@ -0,0 +1,25 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
+  config_get_token: '/^ip access-list <acl_name>$/'
```

Comment:
@mikewiebe I'm not sure that will work right if the nvgen string is `ip` instead of `ipv4`